packages/camera/camera_platform_interface/test/method_channel/method_channel_camera_test.dart

```diff
@@ -1038,6 +1038,52 @@ void main() {
               arguments: <String, Object?>{'cameraId': cameraId}),
         ]);
       });
+
+      test('Should start streaming', () async {
+        // Arrange
```

Comment:
I'm not a big fan of the commenting myself, although I can see the value in reminding people to always think this way in test structure (since it is a good structure, and I've seen plenty of tests that turn into soup when people don't follow it). I was just going for file-level consistency.